src/accelerate/components/List.tsx

```diff
@@ -97,16 +131,39 @@ export default function List( props: Props ) {
 							} }
 						/>
 					</div>
+					<div className="table-add-new">
+						<Dropdown
+							className=""
+							contentClassName=""
+							position="bottom right"
+							renderToggle={ ( { isOpen, onToggle } ) => (
+								<Button
+									variant="primary"
+									onClick={ onToggle }
+									aria-expanded={ isOpen }
+									className='dashicons-before dashicons-plus'
+								>
+									{ __( 'Add New', 'altis' ) }
+								</Button>
+							) }
+							renderContent={ ( { onClose } ) => (
+								<MenuGroup>
+									{/* <MenuItem onClick={ onAddNew( 'block' ) }>
+										{ __( 'Block', 'altis' ) }
+									</MenuItem> */}
+									<MenuItem onClick={ onAddNew( 'post' ) }>
+										{ __( 'Post', 'altis' ) }
+									</MenuItem>
```

Comment:
Not sure I follow why these are using `onClick` and not just regular `<a>` tags

Reply:
Just using core components and that's how it works. Because it's not always navigating away. Happy to drop and check alternatives, it just seemed like a time saver.

Reply:
Hmm, if it's a required prop then I guess it makes sense but it doesn't seem particularly accessible to me